app/code/BodenkoVV/AskQuestion/Ui/Component/Question/DataProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace BodenkoVV\AskQuestion\Ui\Component\Question;
+
+use \BodenkoVV\AskQuestion\Model\ResourceModel\Question\CollectionFactory;
+
+/**
+ * Class DataProvider
+ * @package BodenkoVV\AskQuestion\Ui\Component\Question
+ */
+class DataProvider extends \Magento\Ui\DataProvider\AbstractDataProvider
+{
+    /** @var \BodenkoVV\AskQuestion\Model\ResourceModel\Question\Collection $collection */
+    protected $collection;
+
+    /** @var $loadedData */
+    protected $loadedData;
+
+    /**
+     * DataProvider constructor.
+     * @param $name
+     * @param $primaryFieldName
+     * @param $requestFieldName
+     * @param CollectionFactory $questionCollectionFactory
+     * @param array $meta
+     * @param array $data
+     */
+    public function __construct(
+        $name,
+        $primaryFieldName,
+        $requestFieldName,
+        CollectionFactory $questionCollectionFactory,
```

Comment:
Something is wrong with this class while  setup:di:compile
Please investigate and fix the compilation

Reply:
+